index.js

```diff
@@ -11,12 +11,6 @@ const writeFile = promisify(fs.writeFile);
 
 const isUrl = string => /^(https?|file):\/\/|^data:/.test(string);
 
-const hideElements = elements => {
-	for (const element of elements) {
-		element.style.visibility = 'hidden';
-	}
-};
-
 const removeElements = elements => {
 	for (const element of elements) {
 		element.style.display = 'none';
```

Comment:
@krnik @sindresorhus 

should `removeElements` follows the same approach using `page.addStyleTag`?

Reply:
Yes, I think so but just so the logic is consistent between those two options ;)

Reply:
Yes. I also just realized that we probably should also add `!important` to both of these, so we're sure nothing can override them.